defaultStorage.clear() errors

Using the `.clear()` method returned from makeDefaultStorage errors with:

`global-error-handler.service.ts:13 Error: Uncaught (in promise): NotFoundError: Failed to execute 'objectStore' on 'IDBTransaction': The specified object store was not found.
Error: Failed to execute 'objectStore' on 'IDBTransaction': The specified object store was not found.`


urql version & exchanges:
@urql/core: 2.1.2
@urql/exchange-graphcache: 4.1.2 


Culprit: 

https://github.com/FormidableLabs/urql/blob/276762c030acb22b4138fcd8ca12f851fe04ef2c/exchanges/graphcache/src/default-storage/index.ts#L98-L108

database.transaction should be called with both storeNames:

> const transaction = database.transaction(    [METADATA_STORE_NAME, ENTRIES_STORE_NAME],        'readwrite'      );

[MDN](https://developer.mozilla.org/en-US/docs/Web/API/IDBDatabase/transaction#parameters)


Thanks :)

--- a/.changeset/strange-elephants-add.md
+++ b/.changeset/strange-elephants-add.md
@@ -1,0 +1,5 @@
+---
+"@urql/exchange-graphcache": patch
+---
+
+Fix: add the `ENTRIES_STORE_NAME` to the clear transaction

--- a/exchanges/graphcache/src/default-storage/index.ts
+++ b/exchanges/graphcache/src/default-storage/index.ts
@@ -98,7 +98,7 @@ export const makeDefaultStorage = (opts?: StorageOptions): DefaultStorage => {
     clear() {
       return database$.then(database => {
         const transaction = database.transaction(
-          METADATA_STORE_NAME,
+          [METADATA_STORE_NAME, ENTRIES_STORE_NAME],
           'readwrite'
         );
         transaction.objectStore(METADATA_STORE_NAME).clear();

